assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -79,6 +211,69 @@ describe('Upgrade Landing Controller', function() {
         should.exist(controller);
     });
 
+    describe('when created while prepare is not running', function() {
```

Comment:
"Upgrade Landing Controller when created..." sounds a bit weird... Also, shouldn't this be "before running prepare nodes" instead of "while it's not running"?